component-samples/owner/src/main/java/org/fido/iot/sample/OwnerContextListener.java

```diff
@@ -144,6 +182,8 @@ protected void failed(Exception e) {
     // create tables
     OwnerDbManager manager = new OwnerDbManager();
     manager.createTables(ds);
+    manager.addCustomer(ds, 1, "owner", ownerKeysPem);
+    manager.addCustomer(ds, 2, "reseller", resellerKeysPem);
```

Comment:
reseller => Owner2 .. this would match protocol description.

Reply:
Done